packages/legend-graph/src/models/metamodels/pure/packageableElements/store/relational/mapping/RelationalInputData.ts

```diff
@@ -71,6 +72,10 @@ export class RelationalInputData extends InputData implements Hashable {
     this.data = value;
   }
 
+  setInputType(value: string): void {
```

Comment:
value should be `RelationalInputType`

Reply:
when we are getting the input type via form its a string, so figured we could use this function as it type checks and throws an error , making it more dependable and robust